com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ShaderProgramBuilder.java

```diff
@@ -104,14 +104,15 @@ private boolean getOutputSpirvFlag() {
         return fromProjectOptions || fromProjectProperties;
     }
 
+    private boolean getOutputHlslFlag() {
+        return this.project.option("output-hlsl", "false").equals("true");
+    }
+
     private boolean getOutputWGSLFlag() {
-        boolean fromProjectOptions    = this.project.option("output-wgsl", "false").equals("true");
-        boolean fromProjectProperties = this.project.getProjectProperties().getBooleanValue("shader", "output_wgsl", false);
-        return fromProjectOptions || fromProjectProperties;
+        return this.project.option("output-output_wgsl", "false").equals("true");
```

Comment:
kebab-snake-case?
output-output?

Reply:
Yeah I noticed this and fixed it, thanks 👍